gr-blocks/lib/mute_impl.cc

```diff
@@ -1,6 +1,6 @@
 /* -*- c++ -*- */
 /*
- * Copyright 2004,2010,2013,2018,2019 Free Software Foundation, Inc.
+ * Copyright 2004,2010,2013,2018,2019,2024 Free Software Foundation, Inc.
```

Comment:
ah, while nice, we don't do copyright assignment to the FSF anymore – instead, you retain the copyright to your contributions :-) 

That means you'll want to do something like
```suggestion
 * Copyright 2004,2010,2013,2018,2019 Free Software Foundation, Inc.
 * Copyright 2024 The Gremlin (or their real name)
```